### PR TITLE
Sub-section on structure determination using cryo-EM

### PR DESCRIPTION
**Pull request by lavenderca**
I am a bioinformatics scientist at the National Institute of Environmental Health Sciences (NIEHS). Some colleagues here and I have been following the review and have found it to be a great resource. I decided to add a short section describing recently developed methods for cyro-EM image processing that use deep learning methods. I look forward to participating in this process! Thanks!

### CODE REVIEW

**Comment by agitter** [comment]
Thanks for your interest and contribution @lavenderca. We have a backlog of pull requests now that some of the main authors have temporarily prioritized other projects, but we will definitely respond in detail.

**Comment by agitter** [comment]
@lavenderca this is a fairly self-contained edit so I'm going to work on reviewing it soon.  In preparation, can you please see what merge conflicts have been introduced in `04_study.md`?

To preview my upcoming review, I think we can shorten this to say less about Cryo-EM and keep the emphasis about what it is particularly of interest for deep learning.

File: sections/04_study.md [hunk 1]
```diff
@@ -742,3 +742,51 @@ prioritize specificity over sensitivity, then expert development of
 probabilistic models with hand-developed inputs [@tag:Torracinta2016_sim]. We
 anticipate that these steps will be replaced by deep learning, which will infer
 features simply by its ability to optimize a complex model against data.
+
+### Structure determination and cryo-electron microscopy
```
Comment (agitter) [inline sections/04_study.md]: The data types are very different, but this section seems like it would fit better if it immediately follows the protein structure prediction section.  That way you can include a transition from predicting structure to modeling structure with cryo-EM data.

File: sections/04_study.md [hunk 2]
```diff
@@ -742,3 +742,51 @@ prioritize specificity over sensitivity, then expert development of
...
+
+Recent advances in cryo-electron microscopy (cryo-EM) have allowed the
+structural characterization of proteins and macromolecular complexes at near
+atomic resolution [@doi:10.1016/j.cell.2015.03.049]. Complexes intractable to
+study by established approaches like X-ray crystallography and NMR have been
```
Comment (agitter) [inline sections/04_study.md]: Spell out nuclear magnetic resonance

File: sections/04_study.md [hunk 3]
```diff
@@ -742,3 +742,51 @@ prioritize specificity over sensitivity, then expert development of
...
+For instance, in particle picking, micrographs are scanned to identify
+individual molecular images that will be used in structure refinement. In
+typical applications, hundreds of thousands of particles are necessary to
+determine a structure to near atomic resolution, making manual selection
+impractical [@doi:10.1016/j.cell.2015.03.050]. Furthermore, images containing
+high resolution information are low contrast and difficult to interpret by eye.
+Machine learning approaches have been applied to address these challenges.
+Typical approaches are semi-supervised; a user will select several particles
+manually, and these selections will be used to train a classifier
+[@doi:10.1016/j.jsb.2006.04.006 @doi:10.1016/j.jsb.2014.11.010]. Now
```
Comment (agitter) [inline sections/04_study.md]: After manubot updates, this should now be `[@doi:10.1016/j.jsb.2006.04.006; @doi:10.1016/j.jsb.2014.11.010]`

File: sections/04_study.md [hunk 4]
```diff
@@ -742,3 +742,51 @@ prioritize specificity over sensitivity, then expert development of
...
+convolutional neural networks, such as those applied in facial recognition and
```
Comment (agitter) [inline sections/04_study.md]: Shorten to `Now CNNs are used in DeepPicker`...  We have already introduced CNNs elsewhere.

File: sections/04_study.md [hunk 5]
```diff
@@ -742,3 +742,51 @@ prioritize specificity over sensitivity, then expert development of
...
+image classification [@doi:10.1038/nature14539], are utilized by deep learning
+frameworks DeepPicker [@doi:10.1016/j.jsb.2016.07.006] and DeepEM
+[@doi:10.1186/s12859-017-1757-y] to select particles. In addition to addressing
+shortcomings from manual selection, such as selection bias and poor
+discrimination of low-contrast images, these approaches also provide a means of
+full automation. Not unlike image recognition programs which are initially
```
Comment (agitter) [inline sections/04_study.md]: Here as well I think we can shorten this by removing `Not unlike image recognition programs which are initially trained on large cohorts of references before being applied to independent images [@doi:10.1038/nature14539],`

File: sections/04_study.md [hunk 6]
```diff
@@ -742,3 +742,51 @@ prioritize specificity over sensitivity, then expert development of
...
+images [@doi:10.1038/nature14539], DeepPicker can be trained by reference
+particles from other experiments with structurally unrelated macromolecules,
+allowing for fully automated application to new samples. Subsequent
+classification is fast enough to keep pace with image collection from the
+microscope, allowing for real-time selection of particles as micrographs are
+generated. Downstream of particle picking, deep learning is being applied to
+other aspects of cryo-EM image processing. Statistical manifold learning has
+been implemented in the software package ROME to classify selected particles and
+elucidate the different conformations of the subject molecule necessary for
+accurate 3D structures [@arxiv:1604.04539v2]. These recent tools highlight the
```
Comment (agitter) [inline sections/04_study.md]: This has now been published.  Please switch to [@doi:10.1371/journal.pone.0182130].

File: sections/04_study.md [hunk 7]
```diff
@@ -742,3 +742,51 @@ prioritize specificity over sensitivity, then expert development of
...
 anticipate that these steps will be replaced by deep learning, which will infer
 features simply by its ability to optimize a complex model against data.
+
+### Structure determination and cryo-electron microscopy
+
+Recent advances in cryo-electron microscopy (cryo-EM) have allowed the
+structural characterization of proteins and macromolecular complexes at near
+atomic resolution [@doi:10.1016/j.cell.2015.03.049]. Complexes intractable to
+study by established approaches like X-ray crystallography and NMR have been
+solved using this method [@doi:10.1038/nature12822]. Structures are determined
```
Comment (agitter) [inline sections/04_study.md]: This paragraph reads well, but in the interest in keeping our massive review compact I think we can shorten it.  Can you condense the sentences starting with `Structures are determined` to emphasize what are the areas of need where deep learning is helpful?  My takeaway is that many images are produced rapidly and the analysis requires quick, automated methods to pick particles in real time.

**Comment by lavenderca** [comment]
Thanks, @agitter!  I totally understand the need to keep everything at a reasonable length.  I'll incorporate these changes by the end of the week.

**Comment by dhimmel** [comment]
@agitter did we want to reflow or merge this PR first? This PR is all new content, if I'm not mistaken?

File: sections/04_study.md [hunk 8]
```diff
@@ -474,6 +474,16 @@ summarized above also apply to interfacial contact prediction for protein
...
...; a user will select several particles manually, and these selections will be used to train a classifier [@doi:10.1016/j.jsb.2006.04.006; @doi:10.1016/j.jsb.2014.11.010]. Now CNNs are being used to select particles in tools like DeepPicker [@doi:10.1016/j.jsb.2016.07.006] and DeepEM [@doi:10.1186/s12859-017-1757-y]. In addition to addressing shortcomings from manual selection, such as selection bias and poor discrimination of low-contrast images, these approaches also provide a means of full automation. DeepPicker can be trained by reference particles from other experiments with structurally unrelated macromolecules, allowing for fully automated application to new samples. Downstream of particle picking, deep learning is being applied to other aspects of cryo-EM image processing. Statistical manifold learning has been implemented in the software package ROME to classify selected particles and elucidate the different conformations of the subject molecule necessary for accurate 3D structures [@doi:10.1371/journal.pone.0182130]. These recent tools highlight the general applicability of deep learning approaches for image processing to increase the throughput of high-resolution cryo-EM.
```
Comment (dhimmel) [inline sections/04_study.md]: Can you switch to one line per setence (newlines after every sentence-ending period)?

Comment (lavenderca) [reply sections/04_study.md]: No problem!

File: sections/04_study.md (hunk 1 above)
Comment (lavenderca) [reply sections/04_study.md]: This section was moved and was re-worked to include a transition sentence.

File: sections/04_study.md (hunk 2 above)
Comment (lavenderca) [reply sections/04_study.md]: Spelled out.

File: sections/04_study.md (hunk 7 above)
Comment (lavenderca) [reply sections/04_study.md]: The suggestion was incorporated.  The first paragraph was revised for length.

File: sections/04_study.md (hunk 3 above)
Comment (lavenderca) [reply sections/04_study.md]: Formatting changed.

File: sections/04_study.md (hunk 4 above)
Comment (lavenderca) [reply sections/04_study.md]: Shortened.

File: sections/04_study.md (hunk 5 above)
Comment (lavenderca) [reply sections/04_study.md]: This section was removed.  The second paragraph was also revised for length.

File: sections/04_study.md (hunk 6 above)
Comment (lavenderca) [reply sections/04_study.md]: Citation updated.

**Comment by agitter** [comment]
@lavenderca in my review I said I was going to create a new paragraph starting at `Downstream of particle picking`, but then I realized there are merge conflicts.  Can you please split the paragraph and merge the latest master into your branch so I can merge this?

**Comment by lavenderca** [comment]
Thanks!  Your reviews were very clear, @agitter.
